base/applications/taskmgr/taskmgr.c

```diff
@@ -358,6 +358,10 @@ TaskManagerWndProc(HWND hDlg, UINT message, WPARAM wParam, LPARAM lParam)
         case ID_HELP_ABOUT:
             OnAbout();
             break;
+        // FIXME: MessageBox() must be removed when ReactOS will get HTML Help Support
+        case ID_HELP_TOPICS:
+            MessageBox(hDlg, L"ReactOS HTML Help support isn't implemented yet!", L"Warning", MB_OK | MB_ICONWARNING);
```

Comment:
Instead of duplicating this message box among different programs, I'd create a stub for `winhlp32.exe` and/or `hh.exe` with this message, and make a proper help call from these programs.

Reply:
Neat idea. Though I'm bit novice on Win32 programming, can you give me some guidance on how can I properly implement the stub?